packages/common/src/types/index.ts

```diff
@@ -0,0 +1 @@
+export * from './response.type';
```

Comment:
```suggestion
export * from './response.type';

```
Please add missing trailing line